bots/elasticsearch/bot.py

```diff
@@ -150,6 +257,13 @@ def setup(self):
         logger.debug('Waiting for yellow status')
         self.es_client.cluster.health(wait_for_status='yellow')
 
+        logger.info('Putting Elasticsearch settings')
```

Comment:
Sure. Setting up a new index or updating mappings now requires `--es-setup` given to runbot.